KJ1124/Thor_Puzzle_assign.c

```diff
@@ -0,0 +1,122 @@
+// *******************
+// Power of Thor
+// *******************
+/*
+Use `this link <https://www.codingame.com/ide/puzzle/power-of-thor-episode-1>`_ to access the coding game.
+
+
+Instructions:
+-------------
+
+**Objective:**
+
+Write a program that will guide Thor to reach the light of power in the most efficient manner.
+The light of power is the location that Thor is trying to get to and in order to do this process you will use if statements to create an argument that will move thor in to the placement of the light of power.
+
+
+*Note: You can utilize any coding language that you are most comfortable with to write this code by changing this option in the code tab. I chose to use C++*
+
+*/
+
+#include <iostream>
+#include <string>
+#include <vector>
+#include <algorithm>
+
+using namespace std;
+/*
+Provided Code
+--------------
+*/
+/* These are the functions and variables that are also provided to the developer when you begin the coding game.
+*/
+
+int main() 
+//beginning bracket
+{
+    int light_x; // the X position of the light of power
+    int light_y; // the Y position of the light of power
+    int initial_tx; // Thor's starting X position
+    int initial_ty; // Thor's starting Y position
+    cin >> light_x >> light_y >> initial_tx >> initial_ty; cin.ignore();
+
+
+    while (1) {
+    
+        
+        int remaining_turns; // The remaining amount of turns Thor can move. Do not remove this line.
+        cin >> remaining_turns; cin.ignore();
+        
+/*
+Diagram
+--------
+
+.. image:: InkedThorDiagram.jpg
+
+
+**Discription/Directions:** 
+
+This a diagram of Thor at his starting postion. The red arrows are pointing at the XY plane and the compass below so that you can have a clear orientation of Thor as he moves within the boundary conditions. Remember the objective of the game is to get Thor to move towards the light of power. The directions you will use are: 
+
+*N (North), NE (North East), E (East), SE (South East), S (South), SW(South West), W (West), and NW (North West).*
+
+*/
+/* 
+Thor's Movement
+----------------
+**Based on the position light of power I first determined which direction he would travel and then developed this code on the basis of each test code.**
+
+**Test case 1 & 2:**
```

Comment:
Remove your notes on each test case, and instead focus on how to solve the overall problem. I like instructions on how to run the tests cases, though -- leave that.